### ISSUE TITLE
Removing ARChon

### ISSUE THREAD

**Issue by emm980** [vladikoff/chromeos-apk#171]
What files remain after removing the unpacked extension? I visit pay store and I still get this app is installed message.

**Comment by rocketinventor** [comment]
No files will remain in chrome unless you have loaded an app which was given permission to write to the hard drive. In that case there could be a few empty folders + whatever files that specific app added. Removing Archon should give you a clean uninstall.

The only reason the playstore would say that you have the app is installed on your android, installed the app on your Android in the past, or if you used an apk downloader exstention that worked by tricking Google Play Store into thinking that your browser was you android phone/tablet downloading the App.

In other words, don't worry about it. It just means that at one point in time, Goolge thought you downloaded the app from the Playstore, it is not checking your computer.

**Comment by ViperCode7** [comment]
Haven't. The only playstore interaction in Chrome that I've done was only through ARChon. Removing it did not revert whatever it did to make playstore think I'm on android and even now I get install app choice or app is installed. So somewhere there's still code that interferes.

**Comment by ViperCode7** [comment]
And it's not just on the main profile, I get this throughout Chrome, all I have to do is to login to a Gmail account.

**Comment by vladikoff** [comment]
> I visit play store and I still get this app is installed message.

Could you provide a screenshot? 
You might have to clear your extension cache. Chrome copies / caches some extension sources

**Comment by ViperCode7** [comment]
Will this remove Tampermonkey's installed scripts, ABP's custom filters or other extension's settings?

![screens](https://cloud.githubusercontent.com/assets/6399179/8046030/5ba70860-0e40-11e5-8ed4-2aeaab676f61.png)

**Comment by rocketinventor** [comment]
@ViperCode7 So, what exactly is the problem then?
Did you actually install the Google Play store app on your computer and it thinks it still exists, is that the problem?
If so, you should know that to Google, your device is simply offline. They don't know that it doesn't exists anymore.

**Comment by ViperCode7** [comment]
As I've said enough times, nothing else is installed. No Play store app, no other Android emulator app or anything which could trick the site to see Chrome as an Android Chrome instead of the usual PC Chrome.

**Comment by rocketinventor** [comment]
Based on your screenshots, it doesn't look like anything is wrong. I do not see anything indicating that Google is seeing anything other than usual PC Chrome.
In case you are unfamiliar, the Play Store always gives you an install button to remotely install on devices connected to your account when you are signed in on the computer.
You can rest assured, nothing is installed on or wrong with your computer :smile_cat: :smile: :smiley:

**Comment by ViperCode7** [comment]
OK, I see... so ARChon emulated a device, so that apps got installed, but now that it's gone, the google accounts still keeps track of the emulated devices by reference even if it's inactive. Apps just get abstractly installed, when in fact they're not installed. That seems confusing... but now I'm left with an inactive device which I can't remove from my account, so I'm left with the play store the way it is. Tried to hide it, I still get the install options and it's still listed at devices on which to install.

**Comment by rocketinventor** [comment]
Exactly.
BTW, How did you get the play store app onto Archon? I'd really like to know how you achieved that.
I think you _can_ remove it the device from the list.

**Comment by vladikoff** [comment]
Does it put your PC name into this list: 
![](http://v14d.com/i/5576e9c9df9d4.jpg)
?

**Comment by ViperCode7** [comment]
Don't remember how I did it... but I loaded another Chrome profile in which I never installed this sort of apps or extensions, in there I signed in with a different gmail account and I got the same options to install sans a registered device or the play store app.

On my main account I get only the device, but not my PC. Now I've hidden the devices and I get This app is incompatible with your device under the app title but when I click Install I get No eligible devices in the list. On the clean profile I just get the message You don't have any devices and it points me to the the white shopping bag icon.
